roles/openshift_node/tasks/system_container.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Pre-pull node system container image
```

Comment:
I think either using blocks or breaking this out into two separate task includes would help with the readability here